### DISCUSSION TITLE
ViewerContentToolsProvider -  descriptions are missing

### DISCUSSION THREAD

**Discussion by Paolo-lab** [iTwin/iTwinUI#1560] [Q&A]
Hello everyone, good morning

I recently updated my application to version 3.7.2, in this update the installed version of @itwin/web-viewer-react is ^3.1.4

When I go to use ViewerContentToolsProvider it doesn't show me the descriptions of the features as per the attached image.
 
Furthermore, once I have used a Tool selection, how do I return to the original view?

Thanks for your precious help
Paul
<img width="132" alt="section tool" src="https://github.com/iTwin/iTwinUI/assets/57446652/638d2d2e-bf77-4e91-a748-f6e6282985d0">
<img width="142" alt="Mesure Tool" src="https://github.com/iTwin/iTwinUI/assets/57446652/5a0621a8-8336-4081-829d-affa73e6c488">

**Comment by mayank99** [comment]
This has nothing to do with iTwinUI. Please reach out to [`viewer`](https://github.com/iTwin/viewer) or [`appui`](https://github.com/iTwin/appui)
